animal-sniffer/src/main/java/org/codehaus/mojo/animal_sniffer/ClassFileVisitor.java

```diff
@@ -137,18 +135,20 @@ else if ( ( file.getName().endsWith( ".jar" ) || file.getName().endsWith( ".jmod
      *             or a class file (in which case that single class is processed).
      */
     public void process( Path path )
-            throws IOException {
-        Files.walkFileTree(path, Collections.<FileVisitOption>emptySet(), 10000, new SimpleFileVisitor<Path>() {
+        throws IOException
+    {
+        final SortedSet<Path> files = new TreeSet<>();
+        Files.walkFileTree(path, new SimpleFileVisitor<Path>() {
 
             @Override
-            public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes attrs) throws IOException {
+            public FileVisitResult preVisitDirectory(Path dir, BasicFileAttributes attrs) {
                 return FileVisitResult.CONTINUE;
             }
```

Comment:
Can be removed, the [default implementation](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/nio/file/SimpleFileVisitor.html#preVisitDirectory(T,java.nio.file.attribute.BasicFileAttributes)) behaves the same way.